testcases/PowerNVDump.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/python2
+# IBM_PROLOG_BEGIN_TAG
+# This is an automatically generated prolog.
+#
+# $Source: op-test-framework/testcases/OpTestKernel.py $
+#
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2017
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# IBM_PROLOG_END_TAG
+
+#  @package PowerNVDump.py
+#  This module can contain testcases related to FW & Kernel interaction features
+#  like kdump, fadump and MPIPL etc.
+#
+#   fadump:
+#   The goal of firmware-assisted dump is to enable the dump of
+#   a crashed system, and to do so from a fully-reset system.
+#   For more details refer
+#       https://www.kernel.org/doc/Documentation/powerpc/firmware-assisted-dump.txt
+#
+#   kdump:
+#   Kdump uses kexec to quickly boot to a dump-capture kernel whenever a
+#   dump of the system kernel's memory needs to be taken (for example, when
+#   the system panics).
+#   For more details refer
+#       https://www.kernel.org/doc/Documentation/kdump/kdump.txt
+#
+#   1. Enable both fadump and kdump - trigger a kernel crash
+#   2. Enable only kdump - trigger a kernel crash
+#   3. Disable both - trigger kernel crash
+#   And verify boot progress and collected dump components(vmcore and opalcore)
+#
+
+import time
+import commands
+import re
+import sys
+import pexpect
+
+from common.OpTestConstants import OpTestConstants as BMC_CONST
+from common.OpTestError import OpTestError
+
+import unittest
+import OpTestConfiguration
+from common.OpTestSystem import OpSystemState
+from common.OpTestSSH import ConsoleState as SSHConnectionState
+from common.Exceptions import KernelOOPS, KernelCrashUnknown, KernelKdump, KernelFADUMP, PlatformError, CommandFailed
+
+OPAL_MSG_LOG = "cat /sys/firmware/opal/msglog"
+PROC_CMDLINE = "cat /proc/cmdline"
+OPAL_DUMP_NODE = "/proc/device-tree/ibm,dump/"
+
```

Comment:
As @stewart-ibm suggested I've renamed "ibm,dump" node as "/ibm,opal/dump". Please update this patch.

-Vasant